src/main/fc/rc_adjustments.c

```diff
@@ -741,21 +780,21 @@ void processRcAdjustments(controlRateConfig_t *controlRateConfig)
     }
 
     // Process Absolute adjustments
-    for (int index = 0; index < MAX_ADJUSTMENT_RANGE_COUNT; index++) {
+    for (int index = 0; index < activeAdjustmentRangeCount; index++) {
         static int16_t lastRcData[MAX_ADJUSTMENT_RANGE_COUNT] = { 0 };
```

Comment:
(just use `i` and `int index = activeAdjustmentRangeArray[i]`)

Reply:
Done.